src/sexp.ml

```diff
@@ -5,9 +5,9 @@ include (Usexp : module type of struct include Usexp end
 
 let code_error message vars =
   code_errorf "%a" pp
-    (List (Atom message
+    (List (Quoted_string message
```

Comment:
We should use `atom_or_string` here. Sometimes the message is just a function name, and it looks slightly nicer without the quotes IMO